.github/ISSUE_TEMPLATE/new-tweet-request.yml

```diff
@@ -0,0 +1,32 @@
+name: New Tweet Request
+description: Create a new tweet request
+title: "[Tweet] <title>"
+body:
+  - type: markdown
+    attributes:
+      value: 
+        Thank You for creating a new tweet request and it will be merged soon and the tweet will go out soon
+  - type: checkboxes
+    attributes:
+      options: 
+      - label: Remember to follow the 280 characters limit to tweet 
+        required: true
+  - type: textarea
+    id: Tweet
+    attributes:
+      label: What do you want to tweet?
+      placeholder: Time to drop a tweet.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Why you want to tweet this?
+      placeholder: Tell us what brings you up to write this amazing tweet.
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Detailed information about the tweet.
+      placeholder: Tell us all the information related to the tweet or some amazing examples related to tweet.
+    validations:
+      required: false 
```

Comment:
I think this section should be for any extra notes or thing user would like to tell us. Directly asking detailed info about the tweet feels too much for requesting a simple tweet.

Reply:
That's why this field is not a required field, people can leave it blank if not needed.

Reply:
I'm not able to comprehend "detailed information about the tweet". Wouldn't a person making a tweet request include everything they want to tweet in the tweet itself?
So, I suggested an "extra notes" section for folks to add anything else they want to

Reply:
So should I replace the label with extra notes?

Reply:
I would recommend so, we can have more discussion on discord if needed